<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditCalculator</class>
 <widget class="QDialog" name="creditCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кредитный калькулятор</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>40</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="inputList">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Общая сумма кредита</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Срок (в месяцах)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Процентная ставка</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>121</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="inputDate">
    <item>
     <widget class="QDoubleSpinBox" name="creditSum">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="term">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="interestRate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="outputList">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ежемесячный платеж</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Переплата по кредиту</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Общая выплата</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="outputDate">
    <item>
     <widget class="QLabel" name="monthlyPayment">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="percentSum">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalPayment">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>299</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="creditMode">
    <item>
     <widget class="QRadioButton" name="annuitet">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Аннуитентный</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="diff">
      <property name="text">
       <string>Дифференцированный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
